Common/src/main/java/at/petrak/hexcasting/api/casting/eval/CastingEnvironment.java

```diff
@@ -237,10 +237,10 @@ public boolean isEnlightened() {
      * If there was enough media found, it will return less or equal to zero; if there wasn't, it will be
      * positive.
      */
-    public long extractMedia(long cost) {
+    public long extractMedia(long cost, boolean simulate) {
         for (var extractMediaComponent : preMediaExtract)
```

Comment:
should the onExtractMedia events be called when simulated? perhaps there should be just a separate simulateExtractMedia?

Reply:
I was thinking of having the extractMedia hooks also take a simulate argument

Reply:
oh they do affect cost don't they. yeah that works for me

Reply:
done